Stardrop/ViewModels/MainWindowViewModel.cs

```diff
@@ -575,12 +587,26 @@ private bool ModFilter(object item)
             {
                 return false;
             }
+
             if (!String.IsNullOrEmpty(_filterText) && !String.IsNullOrEmpty(_columnFilter))
             {
                 if (_columnFilter == Program.translation.Get("ui.main_window.combobox.mod_name") && !mod.Name.Contains(_filterText, StringComparison.OrdinalIgnoreCase))
                 {
                     return false;
                 }
+                else if (_columnFilter == Program.translation.Get("ui.main_window.combobox.group"))
+                {
```

Comment:
I am not against having column filter specifically for group names. But I find it not that useful.

You usually do not know exactly what the name of the mod you want to filter for is. What I (and I imagine many of the users) want is to be able to search for a string that appears **either** in group name or mod name, as it was implemented here before.

You will rarely know that now you want to search for group name specifically, but not mod name, and now only the opposite. It can be useful at times, but what I imagine is being able to search for both the group name and mod name **simultaneously**.

This multi-search could be a new filter combobox, if you do not want to have this option by default (which would make sense if we have a separate group name search filter). But then I would like to see a setting to set the default filter search mode (to be able to always open Stardrop and immediately start searching for some string in both the group name and mod name).

The idea (which I had implemented in this PR and expanded upon in next commits intended for future PRs) goes as follows: You search for some name fragment you remember, not exactly sure whether it is in the name of the folder (group name) or in a name of a specific mod (in the manifest file). You expect to see all mods with their groups which either have that name in the manifest file, or in the group name.

What do you think about this functionality? Considering mod authors name the mod folders and mod names in manifest files somewhat inconsistently, it is hard to be satisfied with a search filter for only one of these, or the other, but not both simultaneously.

Reply:
I would prefer each filter to only check what it is labeled for. The suggested change for the "Mod Name" filter checked both the path name and the mod name, which could cause confusion.

Converting the combobox into a checkbox dropdown would be the best solution I think, as it would allow for custom searches like the group and mod name combination.

Having Stardrop remember the last filter used is also a good idea!

Reply:
> Converting the combobox into a checkbox dropdown would be the best solution I think, as it would allow for custom searches like the group and mod name combination.

That is actually a great idea. I never thought about that. That would, together with the last filter cache, solve everything.